### DISCUSSION TITLE
Is there a windows version of the NVIDIA toolkit?

### DISCUSSION THREAD

**Discussion by Ana-Kerlo** [milesial/Pytorch-UNet#425] [Q&A]
Hi, I was hoping to be able to run this on my Windows machine, but I can't figure out how to install the NVIDIA toolkit. Anybody knows if this is possible? 
Thanks

**Comment by milesial** [comment]
I suggest using WSL2 if running windows, you'll get all the supported linux apps

**Comment by ishangodawatta** [comment]
You just need CUDA: https://developer.nvidia.com/cuda-downloads
